### PR TITLE
Kiwigrid logo

### PR DESCRIPTION
**Pull request by jackalski**
kiwigrid(1)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Official Kiwigrid logo in SVG

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6126,6 +6126,11 @@
             "hex": "FD755C",
             "source": "https://kitsu.io/"
         },
+        {
+            "title": "Kiwigrid",
+            "hex": "C7CC00",
+            "source": "https://kiwigrid.com/"
```
Comment (adamrusted) [inline _data/simple-icons.json:6132]:
```suggestion
            "source": "https://kiwigrid.com"
```

File: icons/kiwigrid.svg
Comment (adamrusted) [inline icons/kiwigrid.svg:1]: There seems to be [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/e47295ae-d0a5-49a3-9c9d-328fd36ed7b3) between your PR in black, and the logo available on the site in red. Would you mind taking another look?

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Comment by service-paradis** [comment]
It does not meet our popularity requirements:
https://www.similarweb.com/website/kiwigrid.com/#overview

Do you have more information we can use @jackalski ?

**Comment by service-paradis** [comment]
Closing since it is out of scope.

**Pull request closed without merging**